ai-ml/infrastructure/terraform/fsx-for-lustre.tf

```diff
@@ -7,24 +7,25 @@
 #---------------------------------------------------------------
```

Comment:
Recommend we use the terraform module [here](https://github.com/terraform-aws-modules/terraform-aws-fsx). Example [here](https://github.com/terraform-aws-modules/terraform-aws-fsx/blob/6c39d150edad0047605e4a2c836fde96d448f1a9/examples/lustre/main.tf#L24)